website/cue/reference/components/sources/aws_s3.cue

```diff
@@ -261,7 +261,11 @@ components: sources: aws_s3: components._aws & {
 	telemetry: metrics: {
 		events_in_total:                        components.sources.internal_metrics.output.metrics.events_in_total
 		processed_bytes_total:                  components.sources.internal_metrics.output.metrics.processed_bytes_total
+		component_discarded_events_total:       components.sources.internal_metrics.output.metrics.component_discarded_events_total
```

Comment:
Is this still relevant given the change to emitting `StreamClosedError`?

Reply:
considering @jszwedko proposition [here](https://github.com/vectordotdev/vector/pull/11132#pullrequestreview-873368363), I'll leave it 😉